playbooks/templates/dev-install_net_config.yaml.j2

```diff
@@ -4,17 +4,19 @@ network_config:
 - type: ovs_bridge
   name: br-ex
   use_dhcp: true
-  mtu: 1500
   ovs_extra:
   - br-set-external-id br-ex bridge-id br-ex
   members:
   - type: interface
+{% if host_access_interface is defined %}
+    name: {{ host_access_interface }}
+{% else %}
     name: {{ network_info.public_ipv4.interface }}
+{% endif %}
     primary: true
 - type: ovs_bridge
   name: br-ctlplane
   use_dhcp: false
-  mtu: 1500
```

Comment:
Yeah, I think I just copied these blindly from somewhere. Definitely safer to remove them.